core-java/src/test/java/com/baeldung/stringpool/StringPoolUnitTest.java

```diff
@@ -0,0 +1,43 @@
+package com.baeldung.stringpool;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StringPoolUnitTest {
+
+    @Test
+    public void whenCreatingConstantStrings_thenTheirAddressesAreEqual() {
```

Comment:
Very good test method names! Let's just switch to using AssertJ for assertions instead of JUnit, as per the recently announced policy.

Reply:
Thanks! Just switched in the latest commit, could you please have a look? Thanks very much